iml-api/src/graphql.rs

```diff
@@ -1039,3 +899,37 @@ fn validate_snapshot_name(x: &str) -> Result<(), FieldError> {
         Ok(())
     }
 }
+
+async fn active_mgs_host_fqdn(
+    fsname: &str,
+    pool: &PgPool,
+) -> Result<Option<String>, iml_postgres::sqlx::Error> {
+    let fsnames = &[fsname.into()][..];
+    let maybe_active_mgs_host_id = sqlx::query!(
+        r#"
+            SELECT active_host_id from target WHERE filesystems @> $1 and name='MGS'
+        "#,
```

Comment:
Any reason we can't LEFT JOIN on chroma_core_managedhost? If we could do that it would be a single db call wouldn't it?

Reply:
probably an inner join because we'd want to return None if either of the sides were empty.

That said, this fn was not created in this patch, just moved from another spot in the file. I'm not too keen on changing the query here.

Reply:
sounds good